contracts/registry/TrustedIssuersRegistry.sol

```diff
@@ -182,12 +205,27 @@ contract TrustedIssuersRegistry is ITrustedIssuersRegistry, Ownable {
         for (i = 0; i < claimTopicsLength; i++) {
             trustedIssuerClaimTopics[index][i] = claimTopics[i];
         }
+        delete trustedIssuer[address(trustedIssuers[index])];
```

Comment:
there was a bug on this function as it was not updating `trustedIssuer`. the function `isTrustedIssuer` was not returning the good result after issuer update